packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -484,8 +484,10 @@ class ScaffoldState extends State<Scaffold> {
       List<Widget> bottomSheets = <Widget>[];
       if (_dismissedBottomSheets != null && _dismissedBottomSheets.isNotEmpty)
         bottomSheets.addAll(_dismissedBottomSheets);
-      if (_currentBottomSheet != null)
+      if (_currentBottomSheet != null) {
+        _currentBottomSheet.setState(() { });
```

Comment:
I've removed this setState() call. Will file an issue about making this scenario work.